test/cmocka/src/math/window/CMakeLists.txt

```diff
@@ -8,4 +8,5 @@ cmocka_test(window
 	${PROJECT_SOURCE_DIR}/src/math/base2log.c
 	${PROJECT_SOURCE_DIR}/src/math/decibels.c
 	${PROJECT_SOURCE_DIR}/src/math/exp_fcn.c
+	${PROJECT_SOURCE_DIR}/src/math/exp_fcn_hifi.c
```

Comment:
please squash this patch with previous one, as Marc said, you need get each patch pass all CI separately, here, there is obvious CI error.